CHANGES.rst

```diff
@@ -1,10 +1,15 @@
 Changelog
 =========
 
-Version 0.3.0 [unreleased]
+Version 0.3.0 [2017-05-30]
 --------------------------
 
-WIP
+- `#24 <https://github.com/interop-dev/django-netjsongraph/issues/24>`_:
+  Provided base classes to improve reusability
+- `#27 <https://github.com/interop-dev/django-netjsongraph/issues/27>`_:
+  [link] Added link_status_changed signal
+- `#15 <https://github.com/interop-dev/django-netjsongraph/issues/15>`_:
+   Added ``NETJSONGRAPH_VISUALIZER_CSS`` and instructions to override default templates
```

Comment:
Added ``NETJSONGRAPH_VISUALIZER_CSS`` setting and instructions to override default templates